client/GameState/muncherState.js

```diff
@@ -0,0 +1,42 @@
+  { position: 8, right: 9, left: 7, up: 2, down: 14, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 9, right: 10, left: 8, up: 3, down: 15, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 10, right: 11, left: 9, up: 4, down: 16, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 11, right: null, left: 10, up: 5, down: 17, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 12, right: 13, left: null, up: 6, down: 18, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 13, right: 14, left: 12, up: 7, down: 19, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 14, right: 15, left: 13, up: 8, down: 20, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 15, right: 16, left: 14, up: 9, down: 21, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 16, right: 17, left: 15, up: 10, down: 22, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 17, right: null, left: 16, up: 11, down: 23, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 18, right: 19, left: null, up: 12, down: 24, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 19, right: 20, left: 18, up: 13, down: 25, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 20, right: 21, left: 19, up: 14, down: 26, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 21, right: 22, left: 20, up: 15, down: 27, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 22, right: 23, left: 21, up: 16, down: 28, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 23, right: null, left: 22, up: 17, down: 29, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 24, right: 25, left: null, up: 18, down: 30, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 25, right: 26, left: 24, up: 19, down: 31, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 26, right: 27, left: 25, up: 20, down: 32, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 27, right: 28, left: 26, up: 21, down: 33, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 28, right: 29, left: 27, up: 22, down: 34, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 29, right: null, left: 28, up: 23, down: 35, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 30, right: 31, left: null, up: 24, down: null, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 31, right: 32, left: 30, up: 25, down: null, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 32, right: 33, left: 31, up: 26, down: null, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 33, right: 34, left: 32, up: 27, down: null, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 34, right: 35, left: 33, up: 28, down: null, word: 'tree', phonics: 'ee', occupied: null },
+  { position: 35, right: null, left: 34, up: 29, down: null, word: 'seal', phonics: 'ea', occupied: null },
+];
```

Comment:
Looking at this, I feel like there might be a way to simplify your data structure here? My first thought is a two dimensional array, where you can determine if there's a left or right square by seeing if you're at the beginning or end of the nested array, or top or bottom square by seeing if you're in the first or last array. However, this might make your logic more complex later on, so it's just something to think about.

Reply:
a two dimensional array would actually be pretty cool to make I think. While I did simplify it a little, it is something I want to come back to. I will keep that in mind.